.github/workflows/unit-test.yml

```diff
@@ -23,6 +26,36 @@ jobs:
           TEST_USER: ${{ secrets.PLAYER_TEST_USER }}
       - name: Run Tests
         run: |
-          pnpm test
+          pnpm test:coverage
         env:
           TEST_USER: ${{ steps.json.outputs.encoded }}
+      - name: Report Auth Coverage
+        if: always()
+        uses: davelosert/vitest-coverage-report-action@v2
+        with:
+          name: 'Auth'
+          working-directory: './packages/auth'
```

Comment:
I assume you don't use `working directory: './'` so you get separate reports?

Reply:
I did it because i have no idea what I'm doing 🥇 It was a more convenient way to point to the correct output json.